build-system/test-server.js

```diff
@@ -111,5 +111,17 @@ app.use('/form/post', function(req, res) {
   });
 });
 
+/**
+ * Serve entry point script url
+ */
+app.get('/dist/ww.js', function(req, res, next) {
```

Comment:
we should eventually get rid of this 2nd server running in a different port. This probably can fix your problem: #9561